### ISSUE TITLE
[BUG] Panic on nil attribute value

### ISSUE THREAD

**Issue by tbnguyen1407** [golang-cz/devslog#9]
**Describe the bug**
slog panics on nil attribute value

**To Reproduce**

```
logger := slog.New(devslog.NewHandler(os.Stdout, nil))
logger.Info("info message", "field1", nil)
```

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x98 pc=0x4bcd79]

goroutine 1 [running]:
github.com/golang-cz/devslog.(*developHandler).reducePointerTypeValue(0x58c1b0?, {0x0?, 0x0?}, {0x0?, 0x0?, 0x0?})
	/tmp/gopath25203337/pkg/mod/github.com/golang-cz/devslog@v0.0.4/devslog.go:528 +0x59
github.com/golang-cz/devslog.(*developHandler).colorize(0xc000062180, {0xc00007e000, 0x43, 0x400}, {0xc000016150?, 0x1, 0x0?}, 0x0, {0x5c9100, 0x0, ...})
	/tmp/gopath25203337/pkg/mod/github.com/golang-cz/devslog@v0.0.4/devslog.go:263 +0xef6
github.com/golang-cz/devslog.(*developHandler).processAttributes(0xc000062180, {0xc00007e000, 0x43, 0x400}, 0xc000107b20)
	/tmp/gopath25203337/pkg/mod/github.com/golang-cz/devslog@v0.0.4/devslog.go:201 +0x405
github.com/golang-cz/devslog.(*developHandler).Handle(_, {_, _}, {{0xbab699fc00000000, 0x1, 0x59b7e0}, {0x4e1100, 0xc}, 0x0, 0x4bd17f, ...})
	/tmp/gopath25203337/pkg/mod/github.com/golang-cz/devslog@v0.0.4/devslog.go:115 +0x305
log/slog.(*Logger).log(0xc00007af00, {0x506ae0, 0x5c9100}, 0x0, {0x4e1100, 0xc}, {0xc00007af10, 0x2, 0x2})
	/usr/local/go-faketime/src/log/slog/logger.go:225 +0x1e4
log/slog.(*Logger).Info(...)
	/usr/local/go-faketime/src/log/slog/logger.go:178
main.main()
	/tmp/sandbox809871717/prog.go:13 +0x97

Program exited.
```

**Expected behavior**
No panic. Same behaviour as TextHandler and JsonHandler

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 11

**Additional context**

I just noticed a PR is in-progress https://github.com/golang-cz/devslog/pull/8

**Comment by david-littlefarmer** [comment]
Thank you for report.
Yes i found out this bug earlier, and i am slowly working on it and trying to debug it properly.

**Comment by david-littlefarmer** [comment]
I just released [v0.0.5](https://github.com/golang-cz/devslog/releases/tag/v0.0.5) and this bug should be resolved.
![image](https://github.com/golang-cz/devslog/assets/17728576/12de4899-c304-41ca-b514-c7fc522c1d36)
